sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1132,6 +1132,45 @@ def get_cmis_host_lanes_mask(self, api, appl, host_lane_count, subport):
 
         return host_lanes_mask
 
+   def get_cmis_media_lanes_mask(self, api, appl, lport, subport):
```

Comment:
Can we combine this function with get_cmis_host_lanes_mask?

Reply:
It's doable. But i guess it will be better to have a separate one as it is only happening once after optics insertion.

Reply:
Agreed but was thinking from code reusability perspective since a considerable amount of code is being duplicated in this function.